zebra-state/src/service.rs

```diff
@@ -481,32 +481,32 @@ impl StateService {
         let queued = (checkpoint_verified, rsp_tx);
 
         if self.finalized_block_write_sender.is_some() {
-            // We're still committing finalized blocks
+            // We're still committing checkpoint verified blocks
             if let Some(duplicate_queued) = self
                 .finalized_state_queued_blocks
                 .insert(queued_prev_hash, queued)
             {
                 Self::send_checkpoint_verified_block_error(
                     duplicate_queued,
-                    "dropping older finalized block: got newer duplicate block",
+                    "dropping older checkpoint verified block: got newer duplicate block",
                 );
             }
 
             self.drain_finalized_queue_and_commit();
         } else {
-            // We've finished committing finalized blocks, so drop any repeated queued blocks,
-            // and return an error.
+            // We've finished committing checkpoint verified blocks to finalized queue,
+            // so drop any repeated queued blocks, and return an error.
             //
             // TODO: track the latest sent height, and drop any blocks under that height
             //       every time we send some blocks (like QueuedSemanticallyVerifiedBlocks)
             Self::send_checkpoint_verified_block_error(
                 queued,
-                "already finished committing finalized blocks: dropped duplicate block, \
+                "already finished committing checkpoint verified blocks to finalized queue: dropped duplicate block, \
                  block is already committed to the state",
```

Comment:
```suggestion
                "already finished committing checkpoint verified blocks: dropped duplicate block, \
                 block is already committed to the finalized state",
```

Reply:
We may need to update these checks in `should_restart_sync()`:

https://github.com/ZcashFoundation/zebra/blob/issue6681/zebrad/src/components/sync.rs#L1137
https://github.com/ZcashFoundation/zebra/blob/issue6681/zebrad/src/components/sync.rs#L1171

Reply:
Good catch, I've removed the change to that line, this is meant to be a documentation-only PR.